inference-engine/src/vpu/graph_transformer/src/frontend/frontend.cpp

```diff
@@ -371,14 +367,7 @@ ModelPtr FrontEnd::runCommonPasses(ie::ICNNNetwork& network, const UnsupportedLa
         VPU_LOGGER_SECTION(env.log);
 
         auto convertNetwork = [&convertedNetwork, &originalOrConvertNetwork]() {
-            auto nGraphFunc = originalOrConvertNetwork->getFunction();
-            // Disable shape inference (WA for generic operations)
-            ngraph::op::GenericIE::DisableReshape noReshape(nGraphFunc);
-
-            ngraph::pass::ConvertOpSet3ToOpSet2().run_on_function(nGraphFunc);
-            ngraph::pass::ConvertOpSet2ToOpSet1().run_on_function(nGraphFunc);
-            ngraph::pass::ConvertOpSet1ToLegacy().run_on_function(nGraphFunc);
-            convertedNetwork = InferenceEngine::details::convertFunctionToICNNNetwork(nGraphFunc, *originalOrConvertNetwork);
```

Comment:
@ilya-lavrenov @GlebKazantaev 

Do we use here default approach instead of manual? As @GlebKazantaev already mentioned VPU needs customizable conversion (we're going to insert additional transformations between convert to legacy opset and conversion to CNN)

Reply:
> Do we use here default approach instead of manual

Currently, VPU uses default conversion code (the same as in new ctor of `CNNNetworkImpl`). So, I've decided to replace multiple lines of code using single line. In case of VPU or any other plugin need even more custom approach like you said, all these manual steps are still available. I can revert this particular change for VPU since you are going to change these lines in mean future. OK?

Reply:
reverted this particular VPU change since it's going to be changed in mean future and manual pipeline is needed.

Reply:
Thanks!